core/src/main/java/org/infinispan/util/logging/events/Messages.java

```diff
@@ -121,4 +121,10 @@ public interface Messages {
 
    @Message(value = "Can view statistics via JMX and the `metrics` endpoint.")
    String monitorRoleDescription();
+
+   @Message(value = "Sites connected: %s", id = 100017)
+   String sitesConnected(String sites);
```

Comment:
I think we also need the JGroups site view changes to trigger events https://issues.redhat.com/browse/JDG-7310, but that probably should come in a separate PR.

Reply:
It is already implemented in this PR in the `XSiteEventManagerImpl`. It listens to the JGroups events and sends them to the event log.